docs/book/application-integration/stand-alone.md

```diff
@@ -1,3 +1,7 @@
+<!-- markdownlint-configure-file {
+    "no-inline-html": false
```

Comment:
The rule should not disable for the entire document, only temporarily where the problems arise. The use of HTML is limited to a few cases, so the places should be explicitly marked so that nothing else is added.